interactions/models/internal/listener.py

```diff
@@ -88,12 +93,16 @@ def wrapper(coro: AsyncCallable) -> "Listener":
                 if not name:
                     name = coro.__name__
 
+            # If the coroutine has no parameters, we don't need to pass the event object
+            pass_event_object = len(coro_signature.parameters) != 0
```

Comment:
This doesn't account for `self`

Reply:
It does account for self as callback we are getting is _bound method_ and doesn't have self parameter in the signature

Reply:
I tested it, it doesnt - and no, they arent bound methods proxy

Reply:
They were when the check was in event dispatcher... Welp, fixed now